### PR TITLE
common-osd-overview.rst: add callsign panel and fonts from SD

### PR DESCRIPTION
**Pull request by vierfuffzig**

### CODE REVIEW

**Comment by Hwurzburg** [comment]
Thanks Basti...I will add a delay merge tag, but add the PR link in the Upcoming features section

**Comment by vierfuffzig** [comment]
@Hwurzburg thanks henry. let me do the font file stuff, np. you likely got some more items on your list i guess... ;)

**Comment by vierfuffzig** [comment]
@Hwurzburg added a note paragraph on font handling from filesystem

File: common/source/docs/common-osd-overview.rst [hunk 1]
```diff
@@ -43,7 +43,10 @@ Several font sets are included and can be changed during runtime (and those who
 - ``2`` = Betaflight.  bf- / inav-osd default style
 - ``3`` = Bold. A bolder version of Betaflight
 - ``4`` = Digital. 80's LED clock radio style
- 
+
+.. note::    In firmware 4.1 and above onboard font options might be limited on 1 MB boards to save memory for other features. To make up for that, font upload from file system (SD card) is supported. This allows to choose from up to 10 font files (named "font0.bin" - "font9.bin") placed in the root directory of your SD card. Additionally this allows to easily add individualized and / or customized language charsets to your onboard OSD without the need to compile an individual firmware. Runtime font handling using parameter :ref:`OSD_FONT<OSD_FONT>` remains unaffected. MAVftp might be a convenient option to upload files to your SD card without having to physically remove it from your flight controller.
```
Comment (Hwurzburg) [inline common/source/docs/common-osd-overview.rst:47]:
```suggestion
.. note::    In firmware 4.1 and above, onboard font options might be limited on 1 MB boards to save memory for other features. However, font upload from SD card is supported. This allows the user to choose from up to 10 font files (named "font0.bin" - "font9.bin") placed in the root directory of your SD card. Additionally, this allows the user to easily add individualized and/or customized language charsets for the onboard OSD without the need to compile  individual firmware. Runtime font changing using the parameter :ref:`OSD_FONT<OSD_FONT>` remains unaffected. MAVftp might be a convenient option to upload files to your SD card without having to physically remove it from your flight controller. 
```

**Comment by vierfuffzig** [comment]
thanks @Hwurzburg merged and squashed.

**Comment by Hwurzburg** [comment]
merging for 4.1.....thanks

**Pull request merged**